src/components/MessageList/MessageNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, MouseEventHandler } from 'react';
+
+export interface MessageNotificationProps {
+  children: React.ReactNode;
```

Comment:
can remove this from here as it'll be within the `React.FC` part
```suggestion
```

Reply:
I think I am [facing this issue](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33602) - that's why the explicit children was necessary.